blockly-dom.js

```diff
@@ -291,8 +291,17 @@ Blockly.defineBlocksWithJsonArray([
   },
   {
     type: "get_input_value_with_id",
-    message0: "get the value of the <input> with id %1",
+    message0: "get the %1 value of the <input> with id %2",
```

Comment:
it might be worth creating a new type here, for backward compatibility (everyone's existing code would deserialise the old type, but only the new type would be available -> need to change the toolboxes)